### ISSUE TITLE
Release Cycle

### ISSUE THREAD

**Issue by Veghit** [alk0ve/BitViewer#15]
In order for people to use the program without having to run it on VS we need to upload the EXE as a GitHub release from time to time. Thoughts on release frequency? 
The last one was in March, so monthly or Bi-monthly sounds good to me (if the code has actually been changed, obviously).

**Comment by alk0ve** [comment]
Given that we work on features as needed and don't have any real road-map bi-monthly sounds good.
But I see no real issue with releasing smaller updates whenever we feel like it.

**Comment by Veghit** [comment]
Agreed. Just trying to emphasize that because the only option in GITHUB to upload EXEs is the release one, it should IMHO be used more often than it had been till now.

*BTW, is it possible to commit the latest EXE if we change its extension? say mantaByte.exe.dat or something of the like.
